wordpress-mobile/MediaPicker-iOS
# Issue/230 hide empty albums

Fixes #230 

This doesn't filter completely out by media type, but at least tries to remove albums that are completely empty.

To test:
 - Launch the demo app
 - Open the full screen picker
 - Check that empty albums are not showed
 - Check that is not slow.
